packages/blocs/user_poll/lib/bloc/user_poll_bloc.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:async';
+
+import 'package:bloc/bloc.dart';
+import 'package:meta/meta.dart';
+import 'package:userpoll/userpoll.dart';
+
+import 'user_poll_event.dart';
+import 'user_poll_state.dart';
+export 'user_poll_event.dart';
+export 'user_poll_state.dart';
+
+class UserPollBloc extends Bloc<UserPollEvent, UserPollState> {
+  UserPoll currentPoll;
```

Comment:
Can it be private?

Reply:
Can't, we may change this field during runtime.